invisible_cities/cities/detsim_simulate_electrons.py

```diff
@@ -0,0 +1,52 @@
+import numpy  as np
+
+from typing    import   List, Tuple, Callable
+
+#######################################
+######### ELECTRON SIMULATION #########
+#######################################
+def generate_electrons(energies    : np.array,
+                       wi          : float,
+                       fano_factor : float ) -> np.array:
+    """ generate secondary electrons from energy deposits
+    """
```

Comment:
More info

Reply:
I just noticed this question, sorry for the delay in replying.

I think it explains things better and makes the choice clearer. I think the English could be improved but that's not urgent, we can do a quick edit before merging.